zsh/core.zsh

```diff
@@ -0,0 +1,72 @@
+# Core universal settings that work on all platforms
+
+# PATH
+export PATH=$PATH:$HOME/.local/bin
+
+# History settings
+export HISTSIZE=100000
+export SAVEHIST=100000
+export HISTFILE=~/.cache/zsh/history
+export HISTORY_IGNORE="(ls|cd|pwd|exit|cd)*"
+
+setopt EXTENDED_HISTORY      # Write the history file in the ':start:elapsed;command' format.
```

Comment:
Keep the comments on this hunk. They're good since these are pretty obscure features